pkg/setting/setting.go

```diff
@@ -118,6 +118,7 @@ var (
 	ExternalUserMngInfo     string
 	OAuthAutoLogin          bool
 	ViewersCanEdit          bool
+	EditorsCanOwn           bool
```

Comment:
Please move this property to the Config struct https://github.com/grafana/grafana/blob/master/pkg/setting/setting.go#L199

Reply:
Yes I can do that, care to give some background to why the property should move to Config struct?

Reply:
Ofc! Thanks for asking!

In general, we want to avoid depending on global variables as much as possible. That's why we started moving variables to the struct and pass that struct around instead when possible. Large parts of Grafana codebase is still using global functions and variables so we are migrating one small step at the time :) 

In this case the CFG is injected to HttpServer using a DI framework that we hookup in https://github.com/grafana/grafana/blob/master/pkg/cmd/grafana-server/server.go#L79

Id love to go over more of our backend sometime next week if you want. We started writing down some info about the backend in the [Wiki](https://github.com/grafana/grafana/wiki/Backend:-Architecture-and-future) but it could definitely be improved!

Reply:
Thanks!